website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -37,4 +62,6 @@ The [`run`](/reference/commands/run#the-`--empty`-flag) and [`build`](/reference
 
 ## Quick hits
 
-Coming soon!
\ No newline at end of file
+- [Global config flags](/reference/global-configs/about-global-configs) are deprecated from the [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file and should be moved to the `dbt_project.yml`.
+- A new subcategory of flags has been created for [legacy behaviors](reference/global-configs/legacy-behaviors).
+- The [`--indirect_selection`](/reference/global-configs/indirect-selection) flag used with `dbt test` or `dbt build` configures which tests to run for the nodes you specify.
```

Comment:
maybe an example would help?
```suggestion
- The [`--indirect_selection`](/reference/global-configs/indirect-selection) flag enables you to choose which tests to run for specific nodes when using `dbt test` or `dbt build`.
```

Reply:
We keep these quick hits brief and let the page it links to do the speaking